### ISSUE TITLE
Option to switch off Rig Warning for importer

### ISSUE THREAD

**Issue by OldFlak** [TheGameCreators/GameGuruRepo#4108]
Tis a simple Frame exported as OBJ but seams MAX thinks it's rigged.

![image](https://user-images.githubusercontent.com/12246318/213862814-1e3b05e6-7394-4245-91ef-8af93d0f75f8.png)

**Comment by MonkeyFrogStudio** [comment]
I am not getting this with any of my models that I've been importing into the latest experimental build, and I've been importing quite a few of them lately. 

In what app are you making your models? What format are you exporting from your app and importing into MAX?

**Comment by Granada1** [comment]
With a lot of tools now you have to uncheck the root bone box when exporting your model

**Comment by OldFlak** [comment]
I have been importing models a fair bit over the past week and have not had that message - till yesterday.

I only use Blender and they are just simple OBJ exports.

**Comment by MonkeyFrogStudio** [comment]
Interesting. I'm using Blender, too. I've imported several objects today and still don't get the message. However, I've not been using OBJ, but FBX. If you try FBX for the same model, do you get the same message?

**Comment by synchromesh62** [comment]
@MonkeyFrogStudio 
Didnt Lee do some Blender Rig additions for Bond1's rig in Max ?
Perhaps its related ... could be way off but sure i read something on here.

**Comment by MonkeyFrogStudio** [comment]
@synchromesh62 He did. I am certain it's related due to the message. But since I'm not getting them and you are, and we're both using Blender, I was wondering if it might be related to the file format. I'll try importing an OBJ in a bit on my end to see if I get the message. But I was wondering if the same model that gave you the message would give it to you if it were an FBX you were importing. If it did ... and if I don't get the same message with an OBJ ... then it might indicate something with the model ... possibly something that was slipping through the importer before but isn't now? Not sure ... just trying to help narrow it down. ;)

**Comment by MonkeyFrogStudio** [comment]
@synchromesh62 I just imported an OBJ using the latest experimental build. The OBJ was a model created in Blender and exported as an OBJ. I did not get any message during or after import or even when placing the model in a scene.

**Comment by OldFlak** [comment]
So don't get the message with FBX, but as expected they don't come in properly.
![image](https://user-images.githubusercontent.com/12246318/213897048-de5a5522-2036-48e5-bb9a-bc39be825245.png)

Looks to be a recent bug. Only started getting them yesterday after 2 years or more of importing OBJ.
This is one of the many models I imported over the last month or so. The same model (as with many others) did not get that message previously.
But now it does - weird
![image](https://user-images.githubusercontent.com/12246318/213896767-5182981e-d2f8-4a30-a53e-e78f7c3fa825.png)

**Comment by MonkeyFrogStudio** [comment]
@synchromesh62 For the FBX model, how is it not coming in properly? If you set up your exporter in Blender, they will. The settings for FBX are different than OBJ, btw. What settings are you using when you export FBX?

I agree, it's something that's new due to the changes that Lee has made. However, it may be that the new change "discovered" something that's been in your models all along, but ignored or that's not really important (at least not in the grand scheme of things). The only reason I say this is because any OBJs I import into MAX, created also in Blender, don't cause this message to appear for me in MAX.

**Comment by MonkeyFrogStudio** [comment]
BTW - The two settings with arrows are pretty important when exporting via FBX for use in MAX:

![fbx](https://user-images.githubusercontent.com/36735507/213897165-2e92cb70-412e-4228-889c-348b17c299ed.png)

**Comment by synchromesh62** [comment]
> 

I do not use Blender .. I just commented that i recalled Lee making Blender additions in Max for Bond1's rig and that it may be related to your issue .. All my stuff i make in Sketchup

**Comment by MonkeyFrogStudio** [comment]
@synchromesh62 Sorry. It was OldFlak that stated he uses Blender.

**Comment by MonkeyFrogStudio** [comment]
@OldFlak I have several models with really long names and they import fine (no message, at least with FBX). However, they also use underscores to separate out the name parts, like CS_Sidewalk_Straight_Edge_Half_01. So, I am curious if you use underscores if the error message would still occur for you?

If you are using those settings for your FBX export from Blender, have you also made sure you'd Applied everything (scale, rotation, etc.)? Just checking, I hope you don't mind.

By the way, if you'd like me to test the model on my end, just let me know.

**Comment by OldFlak** [comment]
Earlier on in MAX I used FBX mostly with good results - then Lee brought Assimp into the pictrue and it all went downhill from there, so simply haven't bothered with FBX at all. So use OBJ for static and X for animated stuff like doors etc.

I did previously use underscores but they tend to fluff up in X models (again since Assimp entered the picture) - so went with the lead capital approach. They still get fluffed up with that tho - lol.

`If you are using those settings for your FBX export from Blender, have you also made sure you'd Applied everything (scale, rotation, etc.)? Just checking, I hope you don't mind.`

Not tried that no - probably need to move to current Blender (2.79B here) Just been putting off learning the UI all over again - specially since it was fine in the first place :)

-----

But essentially what it means is they have added a bug where max thinks a long name for a mesh is a bone.....?

**Comment by bond1mjb** [comment]
@OldFlak I exported an OBJ using the same exact filename as yours, using Blender 2.79, and it came into GGMAX just fine with no error message. I you want to attach your obj here I'd be willing to take a look at it also and see what's unique about it.
![blend](https://user-images.githubusercontent.com/87768878/213901856-c48cd942-a4c3-44bd-80e8-79f10d135375.jpg)

![longimport](https://user-images.githubusercontent.com/87768878/213901862-094c62ee-1a3f-4615-b368-d672e9bcc42c.jpg)

**Comment by MonkeyFrogStudio** [comment]
> Earlier on in MAX I used FBX mostly with good results - then Lee brought Assimp into the pictrue and it all went downhill from there, so simply haven't bothered with FBX at all. So use OBJ for static and X for animated stuff like doors etc.

I could be wrong, but I think that Assimp had been a part of MAX since the beginning. However, I do think that he did an update to Assimp? Again, could be wrong here.

In any case, I use exclusively FBX (except for the custom collision meshes, since MAX demands they be OBJ) and get no issues at all. So, it's not Assimp. And has Bond1 pointed out, he used 2.79 of Blender, exported an OBJ with the same long name, and it came into MAX fine with no message about bone names. This would indicate the issue is not with OBJ, Assimp, or MAX, but that something is going on with the model itself. 

If you could post the offending model, either Bond1 or I are willing to look at it and see if we can find the issue ... or at least duplicate the bug.

**Comment by OldFlak** [comment]
@bond1mjb @MonkeyFrogStudio - PM'd you on discord

**Comment by bond1mjb** [comment]
@OldFlak  Hi there OldFlak.   I tested your model, all is fine with the model.  The error message about bones is incorrect - it has to do with spaces in the filename, not bones.   If you remove the spaces, it imports fine without that message.   GGMAX is actually NOT converting your model to a rigged model, I think Lee just has a somewhat badly worded error message warning about spaces in the model name even though it has nothing t
[Canopy_Section_Frame.zip](https://github.com/TheGameCreators/GameGuruRepo/files/10475542/Canopy_Section_Frame.zip)
o do with bones.  If you import this version where I removed the spaces it imports without any error message.  Thanks!
